perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -131,7 +190,11 @@ def _GetOptionDecoderConstructions(cls):
     """
     result = super(CustomMachineTypeSpec, cls)._GetOptionDecoderConstructions()
     result.update({'cpus': (option_decoders.IntDecoder, {'min': 1}),
-                   'memory': (MemoryDecoder, {})})
+                   'memory': (MemoryDecoder, {}),
+                   'gpus': (GceGpuDecoder, {
```

Comment:
Yes, I was planning to add the functionality to add gpus to a named machine type in the next PR. I have a TODO somewhere in this one mentioning that.